### PR TITLE
Translate README to Arabic with Google Translate

### PR DESCRIPTION
**Pull request by fabiosoaresv**
#### What this PR make?
- Translate README to Arabic with help the Google Translate

### CODE REVIEW

**Comment by amrrbakry** [comment]
Hi @fabiosoaresv,

thank you so much for taking the time to tackle this. I just have one suggestion: I think it's better to have a different REAMDME file for the Arabic translation, so maybe create a `README-ar.md` file and leave the current README as is.

I will also make some minor improvements to the translation before merging this in the weekend.

**Comment by fabiosoaresv** [comment]
@amrrbakry sure! I'll make this

**Comment by AmrAdelKhalil** [comment]
Hi @fabiosoaresv , I would like to thank you again for your time for helping in the documentation but there are some words in 
 your Arabic translation that could be enhanced in your PR, I will give a time today to suggest Arabic words for you if you are not familiar with Arabic.

@amrrbakry Do you agree with me at this point?

Thanks again

**Comment by fabiosoaresv** [comment]
@AmrAdelKhalil Hey! Yes, I agree with you. I translated with careful, but I'll be grateful if you could help to improve the translation.

Best regards.

**Comment by amrrbakry** [comment]
@AmrAdelKhalil Yes, I completely agree. I was going to work on this, but I haven't had the time just yet. If you can help with it, that would be really helpful. Thanks.

File: README-ar.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
+[![CircleCI](https://circleci.com/gh/fawry-api/fawry.svg?style=svg)](https://circleci.com/gh/fawry-api/fawry)
+
+# Fawry
+
+**تنصل:** نحن لسنا تابعين رسميًا لشركة فوري.
+
+مكتبة التوصيل والتشغيل التي تجعل التعامل مع بوابة الدفع الخاصة بـ Fawry أمرًا سهلاً:
```
Comment (AmrAdelKhalil) [inline README-ar.md:7]:
```suggestion
مكتبة لتسهيل التعامل مع خدمات الدفع الخاصة ب فوري:
```

File: README-ar.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+- [اشحن العملاء](https://github.com/fawry-api/fawry#charge-customers)
```
Comment (AmrAdelKhalil) [inline README-ar.md:9]:
```suggestion
- [لإجراء عملية شحن](https://github.com/fawry-api/fawry#charge-customers)
```

File: README-ar.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+- [العملاء برد](https://github.com/fawry-api/fawry#refund-customers)
```
Comment (AmrAdelKhalil) [inline README-ar.md:10]:
```suggestion
- [لإجراء عملية استرداد](https://github.com/fawry-api/fawry#refund-customers)
```

File: README-ar.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+- [احصل على حالة الدفع](https://github.com/fawry-api/fawry#get-payment-status)
```
Comment (AmrAdelKhalil) [inline README-ar.md:11]:
```suggestion
- [حالة الدفع](https://github.com/fawry-api/fawry#get-payment-status)
```

File: README-ar.md [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+- [تحليل رد خدمة فوري V2](https://github.com/fawry-api/fawry#parse-fawry-service-callback-v2)
+- [مفاتيح التكوين كمتغيرات البيئة](https://github.com/fawry-api/fawry#configuration-keys-as-environment-variables)
```
Comment (AmrAdelKhalil) [inline README-ar.md:13]:
```suggestion
- [طلب البيانات الخاصة بإعدادات الإستخدام](https://github.com/fawry-api/fawry#configuration-keys-as-environment-variables)
```

File: README-ar.md [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+
+_يتم دعم بيئات الإنتاج وصندوق الحماية الخاص بـ فوري_
```
Comment (AmrAdelKhalil) [inline README-ar.md:15]:
```suggestion
_المكتبة تدعم النظام التجريبي ايضا_
```

File: README-ar.md [hunk 7]
```diff
@@ -0,0 +1,136 @@
...
+
+## التركيب
```
Comment (AmrAdelKhalil) [inline README-ar.md:17]:
```suggestion
## لإضافة وتشغيل المكتبة
```

File: README-ar.md [hunk 8]
```diff
@@ -0,0 +1,136 @@
...
+
+ثم نفذ:
+
+    $ bundle
+
+أو قم بتثبيته بنفسك على النحو التالي:
+
+    $ gem install fawry
+
+## إستعمال
```
Comment (AmrAdelKhalil) [inline README-ar.md:33]:
```suggestion
## طريقة الإستعمال
```

File: README-ar.md [hunk 9]
```diff
@@ -0,0 +1,136 @@
...
+
+### اشحن العملاء
```
Comment (AmrAdelKhalil) [inline README-ar.md:35]:
```suggestion
### لإجراء عملية شحن
```

File: README-ar.md [hunk 10]
```diff
@@ -0,0 +1,136 @@
...
+           "customer_mobile": '012345678901',
+           "payment_method": 'PAYATFAWRY',
+           "currency_code": 'EGP',
+           "amount": 20,
+           "fawry_secure_key": 'fawry_secure_key',
+           "description": 'the charge request description',
+           "charge_items": [{ "item_id": 'fk3fn9flk8et9a5t9w3c5h3oc684ivho',
+                              "description": 'desc', "price": 20, "quantity": 1 }] }
+
+# استخدم خيار وضع الحماية للاتصال بصندوق الحماية الخاص بفوري
```
Comment (AmrAdelKhalil) [inline README-ar.md:50]:
```suggestion
# استخدم خيار النظام التجريبي للاتصال بالنظام التجريبي الخاص بفوري
```

File: README-ar.md [hunk 11]
```diff
@@ -0,0 +1,136 @@
...
+#  => #<Fawry::FawryResponse:0x0000564257d0ea90 @type="ChargeResponse", @reference_number="931600239",
+#                                               @merchant_ref_number="io5jxf3jp27kfh8m719arcqgw7izo7db",
+#                                               @expiration_time=1573153206979, @status_code=200,
+#                                               @status_description="Operation done successfully">
+
+res.success? # => true
+res.reference_number # => 931600239
+```
+
+### العملاء برد
```
Comment (AmrAdelKhalil) [inline README-ar.md:61]:
```suggestion
### لإجراء عملية استرداد
```

File: README-ar.md [hunk 12]
```diff
@@ -0,0 +1,136 @@
...
+           "fawry_secure_key": 'fawry_secure_key' }
+
+res = Fawry.refund(params, sandbox: true)
+#  => #<Fawry::FawryResponse:0x0000564257d0ea90 @type="ResponseDataModel", @status_code=200,
+#                                               @status_description="Operation done successfully">
+
+res.success? # => true
+```
+
+### احصل على حالة الدفع
```
Comment (AmrAdelKhalil) [inline README-ar.md:76]:
```suggestion
### حالة الدفع
```

File: README-ar.md [hunk 13]
```diff
@@ -0,0 +1,136 @@
...
+fawry_callback = Fawry.parse_callback(callback_params, {})
+# <Fawry::FawryCallback:0x000056339ac43730 @request_id="c72827d084ea4b88949d91dd2db4996e", @fawry_ref_number="970177",
+#                                          @merchant_ref_number="9708f1cea8b5426cb57922df51b7f790", @customer_mobile="01004545545",
+#                                          @customer_mail="fawry@fawry.com", @order_status="NEW", @order_amount=150.0, @fawry_fees=2.0, ...>
+
+fawry_callback.fawry_ref_number # => 970177
+fawry_callback.order_status # => NEW
+```
+
+### مفاتيح التكوين كمتغيرات البيئة
```
Comment (AmrAdelKhalil) [inline README-ar.md:117]:
```suggestion
### طلب البيانات الخاصة بإعدادات الإستخدام
```

File: README-ar.md [hunk 14]
```diff
@@ -0,0 +1,136 @@
...
+
+يمكن إرسال مفاتيح تهيئة فوري مثل رمز التاجر ومفتاح الأمان مع المعلمات (`merchant_code`, `fawry_secure_key` ) الى **charge**, **refund**, **payment_status** طرق, _أو_ يمكن تعيينها كمتغيرات البيئة: (`FAWRY_MERCHANT_CODE`, `FAWRY_SECURE_KEY`).
```
Comment (AmrAdelKhalil) [inline README-ar.md:119]:
```suggestion
يمكن إرسال بيانات تهيئة فوري مثل رمز التاجر ومفتاح الأمان خلال البيانات المعطاه (`merchant_code`, `fawry_secure_key` ) الى **charge**, **refund**, **payment_status** طرق, _أو_ يمكن تعيينها كمتغيرات لنظام التشغيل: (`FAWRY_MERCHANT_CODE`, `FAWRY_SECURE_KEY`).
```

File: README-ar.md [hunk 15]
```diff
@@ -0,0 +1,136 @@
...
+
+لتحليل fawry معاودة الاتصال ، يجب عليك ضبط متغير البيئة `FAWRY_SECURE_KEY`.
+
+## لكى يفعل:
```
Comment (AmrAdelKhalil) [inline README-ar.md:123]:
```suggestion
## الخطوات القادمة المطلوب تنفيذها:
```

File: README-ar.md [hunk 16]
```diff
@@ -0,0 +1,136 @@
...
+- أضف وثائق API العامة إلى README
+- إضافة خيار لرفع الاستثناء عند فشل الطلب
+
+## تطوير
```
Comment (AmrAdelKhalil) [inline README-ar.md:127]:
```suggestion
## تطوير المكتبة
```

File: README-ar.md [hunk 17]
```diff
@@ -0,0 +1,136 @@
...
+
+بعد التحقق من الريبو ، قم بتشغيل `bin/setup` لتثبيت التبعيات.
+ثم اركض `rake spec` لإجراء الاختبارات. يمكنك أيضا الجري `bin/console` للمطالبة التفاعلية التي تسمح لك بالتجربة.
```
Comment (AmrAdelKhalil) [inline README-ar.md:130]:
```suggestion
ثم نفذ الأمر `rake spec` لإجراء الاختبارات. يمكنك أيضا الجري `bin/console` للمطالبة التفاعلية التي تسمح لك بالتجربة.
```

File: README-ar.md [hunk 18]
```diff
@@ -0,0 +1,136 @@
...
+
+لتثبيت هذه الأحجار الكريمة على جهازك المحلي ، قم بتشغيل `bundle exec rake install`. لإصدار إصدار جديد ، قم بتحديث رقم الإصدار بتنسيق `version.rb`, ثم ركض `bundle exec rake release`, الذي سينشئ علامة git للإصدار ، ويدفع التزامات git والعلامات ، ويدفع ملف `.gem` يا صديق [rubygems.org](https://rubygems.org).
```
Comment (AmrAdelKhalil) [inline README-ar.md:132]:
```suggestion
لتثبيت هذه المكتبة على جهازك ، قم بتشغيل `bundle exec rake install`. لإصدار إصدار جديد ، قم بتحديث رقم الإصدار بتنسيق `version.rb`, ثم نفذ الأمر `bundle exec rake release`, الذي سينشئ علامة git للإصدار ، ويدفع التزامات git والعلامات ، ويدفع ملف `.gem` يا صديق [rubygems.org](https://rubygems.org).
```

File: README-ar.md [hunk 19]
```diff
@@ -0,0 +1,136 @@
+[![CircleCI](https://circleci.com/gh/fawry-api/fawry.svg?style=svg)](https://circleci.com/gh/fawry-api/fawry)
+
+# Fawry
+
+**تنصل:** نحن لسنا تابعين رسميًا لشركة فوري.
```
Comment (AmrAdelKhalil) [inline README-ar.md:5]:
```suggestion
**تنويه:** نحن لسنا تابعين رسميًا لشركة فوري.
```

**Review by AmrAdelKhalil — COMMENTED** [review]
Hi Guys, I tried my best to help in updating the translation, Could you check again please @amrrbakry ?
Thank you guys

File: README-ar.md [hunk 20]
```diff
@@ -0,0 +1,136 @@
...
+```
+
+### مفاتيح التكوين كمتغيرات البيئة
+
+يمكن إرسال مفاتيح تهيئة فوري مثل رمز التاجر ومفتاح الأمان مع المعلمات (`merchant_code`, `fawry_secure_key` ) الى **charge**, **refund**, **payment_status** طرق, _أو_ يمكن تعيينها كمتغيرات البيئة: (`FAWRY_MERCHANT_CODE`, `FAWRY_SECURE_KEY`).
+
+لتحليل fawry معاودة الاتصال ، يجب عليك ضبط متغير البيئة `FAWRY_SECURE_KEY`.
+
+## لكى يفعل:
+- أضف وثائق API العامة إلى README
```
Comment (amrrbakry) [inline README-ar.md:124]:
```suggestion
```

**Review by amrrbakry — APPROVED** [review]

**Comment by amrrbakry** [comment]
@fabiosoaresv @AmrAdelKhalil Merged! Thank you so much, guys! 👏 👏

**Pull request merged**